default/AI/charting/charts.py

```diff
@@ -20,7 +20,7 @@
 from glob import glob
 import traceback
 
-dataDir = os.environ.get('HOME', "") + "/.freeorion"
+dataDir = (os.environ.get('HOME', "") + "/.freeorion" ) if os.name != 'posix' else (os.environ.get('XDG_DATA_HOME', os.environ.get('HOME', "") + "/.local/share") + "/freeorion") 
```

Comment:
charts.py appears to be a stand-alone charting app, that parses the AI logs.  It has no dependencies on any freeOrion code.  I didn't want to introduce new dependencies.